index.html

```diff
@@ -0,0 +1,180 @@
+                        <article>
+                            <figure>
+                                <a href="./pages/la_note_enchantee.html">
+                                    <img 
+                                        src="./assets/images/restaurants/stil-u2Lp8tXIcjw-unsplash.jpg" 
+                                        alt="Assiette de boulettes de viandes et tomates">
+                                    <span>Nouveau</span>
+                                </a>
+                            </figure>
+                                <div>
+                                    <a href="./pages/la_note_enchantee.html">
+                                        <h3>La note enchantée</h3>
+                                        <p>Charonne</p>
+                                        <!--add icône-->
+                                    </a>
+                                </div>
+                        </article>
+                        <article>
+                            <figure>
+                                <a href="./pages/a_la_francaise.html">
+                                    <img 
+                                        src="./assets/images/restaurants/toa-heftiba-DQKerTsQwi0-unsplash.jpg" 
+                                        alt="Assiette fantaisie en gros plan">
+                            </a>
+                            </figure>
+                                <div>
+                                    <a href="./pages/a_la_francaise.html">
+                                        <h3>À la française</h3>
+                                        <p>Cité Rouge</p>
+                                        <!--add icône-->
+                                    </a>
+                                </div>
+                        </article>
+                        <article>
+                            <figure>
+                                <a href="./pages/le_delice_des_sens.html">
+                                    <img 
+                                        src="./assets/images/restaurants/louis-hansel-shotsoflouis-qNBGVyOCY8Q-unsplash.jpg" 
+                                        alt="Assiette de poisson et ses petits légumes">
+                                </a>
+                            </figure>
+                                <div>
+                                    <a href="./pages/le_delice_des_sens.html">
+                                        <h3>Le délice des sens</h3>
+                                        <p>Folie-Méricourt</p>
+                                        <!--add icône-->
+                                    </a>
+                                </div>
+                        </article>
+                    </div>
+            </section>     
+        <!--main content-->
+        </main>
+        <footer>
+        <!--footer-->
+            <div>
+                <nav>
+                <h3>Ohmyfood</h3>
+                    <div>
```

Comment:
remove this div

Reply:
i removed this div before h3, not after.